### PR TITLE
#160451452 setup sass 

### PR DESCRIPTION
**Pull request by onwuvic**
#### What does this PR do?
It setup sass processor

#### Description of Task to be completed?
- install SASS loader, CSS loader, and style loader
- Configure the loaders on Webpack
- create a SASS style file

#### How should this be manually tested?
- open the react app
- the text color has a deep blue color style with SASS.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
Story type: chore
Story title: Setup SASS processing on the client
Story ID: #160451452

#### Screenshots (if appropriate)
N/A
#### Questions:```
N/A

### CODE REVIEW

File: src/styles/style.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$themeColor: #06576D;
+
+.theme-color {
+    color: $themeColor;
+}
```
Comment (udochukwu) [inline src/styles/style.scss]: Please, Add a new line at the file ending.

**Review by udochukwu — DISMISSED** [review]
LGTM

**Review by slimsolz — COMMENTED** [review]
LGTM

**Review by s-akinrele — APPROVED** [review]
GTM

File: package.json [hunk 2]
```diff
@@ -32,8 +32,12 @@
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.4",
     "babel-preset-env": "^1.7.0",
+    "css-loader": "^1.0.1",
     "eslint-plugin-react": "^7.11.1",
     "html-webpack-plugin": "^3.2.0",
```
Comment (onwuvic) [inline package.json:37]: I added the necessary loaders.

File: src/index.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { render } from 'react-dom';
+import './styles/style.scss';
 
 const Index = () => (
     <div>
-      <h1>Hello! Welcome to Authors Haven</h1>
+      <h1 className="theme-color">Hello! Welcome to Authors Haven</h1>
```
Comment (onwuvic) [inline src/index.js:7]: I added the class style from the sass file.

**Pull request merged**